### DISCUSSION TITLE
Missing library when building

### DISCUSSION THREAD

**Discussion by mattthhh** [i3/i3#5571] [Q&A]
Hello I try to build i3 but im not familiar with meson, and this is what meson says :
```
../meson.build:328:12: ERROR: C shared or static library 'ev' not found
```
I know that I have the library installed in `$HOME/.local/`, because I have no access to `sudo` and `apt`. I have some env vars :
```
export PATH=$HOME/.local/bin:$PATH
export LD_LIBRARY_PATH=$HOME/.local/lib:$LD_LIBRARY_PATH
export PKG_CONFIG_PATH=$HOME/.local/lib/pkgconfig:$PKG_CONFIG_PATH
export C_INCLUDE_PATH=$HOME/.local/include
```
but it still don't find the library, any idea ?

**Comment by aspauldingcode** [comment]
I have the same problem...
